packages/components/src/components/input/input.scss

```diff
@@ -253,4 +253,8 @@
     &__wrapper {
         margin-bottom: 1.6rem;
     }
+    &__token-input {
```

Comment:
Can we use more global naming? like input-group?

Reply:
Yes, changed relevant styles.